package.json

```diff
@@ -28,7 +28,6 @@
     "@radix-ui/react-popover": "^0.1.6",
     "@radix-ui/react-radio-group": "^0.1.5",
     "@radix-ui/react-select": "0.1.1",
-    "@self.id/core": "0.1.0",
```

Comment:
@levity any reason you didn't remove the package as well? I went ahead and removed this... in https://github.com/coordinape/coordinape/pull/1532/commits/215bc5e088cb49676cfe34d72ab3fc4ab7a5ad3d

Reply:
Good catch!